kafka-rest/src/main/java/io/confluent/kafkarest/response/UrlFactory.java

```diff
@@ -22,14 +22,26 @@
 
   /**
    * Returns an absolute URL to the resource/collection identifiable by the given path {@code
-   * components}.
+   * segments}.
    *
-   * <p>The URL will be in the form {@code <schema>://<authority>/<base_path>/<components>}, where
+   * <p>The URL will be in the form {@code <schema>://<authority>/<base_path>/<segments>}, where
```

Comment:
It's a bit confusing because what is being produced is actually a URI string, not an URL string. I'm going to refactor later to change UrlFactory and UrlBuilder to UriFactory and UriBuilder, and for them to return actual URIs.